### ISSUE TITLE
How to specify aging parameters during inference?

### ISSUE THREAD

**Issue by Ruize-Ma** [HasnainRaz/Fast-AgingGAN#31]
Hi,

I am using the provided Generator class to infer aging effects on images. However, I couldn't find a clear way to input or specify the desired aging parameter (e.g., target age) during the inference process.

From my understanding, the Generator currently takes only the image tensor as input, but it lacks a mechanism to control the degree or target of aging transformation. Could you clarify:

Is there a way to input a parameter (e.g., a target age or aging factor) during inference to control the aging effect?
If not, what would be the recommended modification to the Generator or ResidualBlock to include such a parameter?
Any guidance or example implementation would be greatly appreciated!

Thank you for your time and support.

**Comment by HasnainRaz** [comment]
Yes, that is correct. It is not possible to control the strength of the aging using this method. As this is based on a very old CycleGAN framework. You would probably be better off doing class & image conditional generation using one of the few modern methods like stylegan or diffusion models.
